app/src/ui/lib/radio-group.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { RadioButton } from './radio-button'
+
+interface IRadioGroupProps<T> {
+  /** The id of the element that serves as the menu's accessibility label */
+  readonly ariaLabelledBy?: string
+
+  /**
+   * The currently selected item, denoted by its key.
+   */
+  readonly selectedKey: T
+
+  /**
+   * The keys of the radio buttons to display, in order of the radio buttons.
+   */
+  readonly radioButtonKeys: ReadonlyArray<T>
```

Comment:
This "in order of the radio buttons" doesn't feel happy to me since if you provide them in a different order than you provide the children to the `RadioGroup` then the radio buttons will behave weird about what is being checked. Granted one bout of testing should highlight that. The reason I did this was because I really like the straightforward readability of using the children in the `RadioGroup` as the labels for `RadioButtons` built in this component. If feels akin to if I were to do something like `<div role="radioGroup"><radio button><radio button></div>`; therefore easier to digest (especially for simple radio groups that just have text in the labels). 

An alternative pattern is to do like list row component and have a "renderLabel" prop that passes the key up. This would be more deterministic; and if anyone feels strongly that we should use that pattern I am open to changing it.

Reply:
I'm a bit torn about this approach. I get what you mean about readability, but I also find it confusing because it doesn't reflect the actual hierarchy in the DOM after rendering. I do see why you did it, since it's not like a homogeneous list of items what we render, so using the `renderWhatever` approach is a bit more annoying, but I also think it's more React if that makes sense.

I think I prefer a `renderWhatever` approach, but I won't block the PR on that.

Reply:
Done. Easy enough to change, and I was torn too so your vote for other approach was plenty enough to push me over. :D